### ISSUE TITLE
Feature Request: Command Palette

### ISSUE THREAD

**Issue by AlexMason** [heyman/heynote#8]
Add a visual command palette similar to what is offered in VS code. A proper command palette can provide a layer of discoverability for someone to find an obscure or less commonly used command/key bind.

![image](https://github.com/heyman/heynote/assets/740625/079f05cc-d608-4e34-a661-eaecbbe532f7)

**Comment by kjk** [comment]
FYI: I've implemented command palette in my fork. It looks like this:

![image](https://github.com/heyman/heynote/assets/7935/adf01894-dc59-4ba0-9472-f5be2fbfc623)

My fork: https://github.com/kjk/edna, https://edna.arslexis.io/

**Comment by heyman** [comment]
Yes, a command panel is planned - together with proper key bindings support.
